_extensions/seedcase-project/seedcase-theme/glossary.json

```
@@ -0,0 +1,18 @@
+[
+  {
+    "item": "data",
+    "description": "Data can mean any piece of information that someone would like to use to answer questions. What is considered data (and metadata) is highly dependent on people and what they intend to do with information that is collected. In the context of Seedcase Sprout, data is any information collected for the purposes of doing analyses on them to answer questions. An example might be data collected from people participating in a study on health and disease. "
```

```suggestion
    "description": "Data can mean any piece of information that someone would like to use to answer questions. What is considered data (and metadata) is highly dependent on people and what they intend to do with information that is collected. In the context of Seedcase Sprout, data is any information collected for the purposes of doing analyses on them to answer questions. An example might be data collected from people participating in a study on health and disease."
```
Or it is intentional that you have a space after all `description`s?